src/layouts/navbar/style.css

```diff
@@ -0,0 +1,41 @@
+.navbar {
+  display: flex;
+  align-items: center;
+  background-color: #000000;
```

Comment:
why this color , is used directly create this a global if you are using this color twice